website/docs/r/dbaas_postgresql_extension_v1.html.markdown

```diff
@@ -1,124 +1,68 @@
 ---
 layout: "selectel"
-page_title: "Selectel: selectel_dbaas_extension_v1"
-sidebar_current: "docs-selectel-resource-dbaas-extension-v1"
+page_title: "Selectel: selectel_dbaas_postgresql_extension_v1"
+sidebar_current: "docs-selectel-resource-dbaas-postgresql-extension-v1"
 description: |-
-  Manages a V1 extension resource within Selectel Managed Databases Service.
+  Creates and manages a PostgreSQL extension in Selectel Managed Databases Service using public API v1.
 ---
 
 # selectel\_dbaas\_extension\_v1
 
-**WARNING**: This resource is deprecated. You should use extension resource for specific datastore type.
-
 Manages a V1 extension resource within Selectel Managed Databases Service. Can be installed only for PostgreSQL datastores.
 
+Creates and manages a PostgreSQL extension using public API v1. Applicable to PostgreSQL, PostgreSQL for 1C, and PostgreSQL TimescaleDB datastores. For more information about Managed Databases, see the official Selectel documentation for [PostgreSQL](https://docs.selectel.ru/cloud/managed-databases/postgresql/), [PostgreSQL for 1C](https://docs.selectel.ru/cloud/managed-databases/postgresql-for-1c/), and [PostgreSQL TimescaleDB](https://docs.selectel.ru/cloud/managed-databases/timescaledb/).
+
```

Comment:
For more information about extensions, see the official Selectel documentation for [PostgreSQL](https://docs.selectel.ru/cloud/managed-databases/postgresql/add-extensions/), [PostgreSQL for 1C](https://docs.selectel.ru/cloud/managed-databases/postgresql-for-1c/extensions-1c/), and [PostgreSQL TimescaleDB](https://docs.selectel.ru/cloud/managed-databases/timescaledb/add-extensions/).